### PR TITLE
[RHINENG-13823] Include unit tests in konflux pull_request

### PR DESCRIPTION
**Pull request by mshriver**
The configuration will need tuning to set resource requests. 

Resource limits pulled from https://github.com/RedHatInsights/compliance-frontend/pull/2217/commits/67e45519714010d5eb524f8af5b17afd31741005

### CODE REVIEW

**Comment by avisiedo** [comment]
@mshriver  Build is failing with:

```
subprocess exited with status 1
subprocess exited with status 1
Error: building at STEP "RUN npm install": exit status 1
```

On `build-container` task; make me think related with this: https://github.com/podengo-project/idmsvc-frontend/blob/main/.tekton/idmsvc-frontend-pull-request.yaml#L346

But unsure the root cause.

File: deploy/frontend.yaml [hunk 1]
```diff
@@ -46,6 +46,10 @@ parameters:
   - name: IMAGE_TAG
     required: true
   - name: IMAGE
+<<<<<<< HEAD
     value: quay.io/redhat-services-prod/insights-management-tenant/insights-idmsvc/idmsvc-frontend
+=======
+    value: "redhat-services-prod/insights-management-tenant/insights-idmsvc/idmsvc-frontend"
+>>>>>>> 4faa71f (Move ClowdApp to Konflux image)
```
Comment (avisiedo) [inline deploy/frontend.yaml:53]:
```suggestion
    value: quay.io/redhat-services-prod/insights-management-tenant/insights-idmsvc/idmsvc-frontend
```

**Review by avisiedo — CHANGES_REQUESTED** [review]
@mshriver  thanks for the change

The pipeline seems to work with the last execution.

Only I observed one suggested change, after that, it is a LFTM

**Comment by frasertweedale** [comment]
I fixed that merge issue and rebased.

**Review by frasertweedale — APPROVED** [review]

**Pull request merged**